docs/panel/getting-started.mdx

```diff
@@ -41,8 +41,8 @@ Pelican runs on a wide range of operating systems, so pick whichever you are mos
     ```
 </Admonition>
 
-* PHP `8.2` with the following extensions: `gd`, `mysql`, `mbstring`, `bcmath`, `xml`, `curl`, `zip`, `intl`, `sqlite3` and `fpm`
-* `sqlite3` **or** MySQL `8` (`mysql-server`)
+* PHP `8.2` (recommended) or `8.3` with the following extensions: `gd`, `mysql`, `mbstring`, `bcmath`, `xml`, `curl`, `zip`, `intl`, `sqlite3` and `fpm`
+* SQLite `3.35.0`+ (`sqlite3`) **or** MySQL `8` (`mysql-server`) **or** MariaDB `10.3`+
```

Comment:
Sqlite3 isn't required to be installed for sqlite to work, only the php extension is required.

Reply:
We should still list as potential dependency so people see that it's supported and have it as helper tool installed.